ui_framework/components/menu/menu.js

```diff
@@ -0,0 +1,39 @@
+import React, {
+  PropTypes,
+} from 'react';
+import classNames from 'classnames';
+
+const getClassName = ({ className, contained = false }) =>
+  classNames('kuiMenu', className, {
+    'kuiMenu--contained': contained
+  });
```

Comment:
Minor nit: would you mind adding the braces and `return` keyword here? I don't like adding unnecessary syntax either, but I think it will help readability because this will be consistent with how we format functions (fat arrow and otherwise) elsewhere.

```javascript
const getClassName = ({ className, contained = false }) => {
  return classNames('kuiMenu', className, {
    'kuiMenu--contained': contained,
  });
};
```